pyslackersweb/settings.py

```diff
@@ -3,7 +3,7 @@
 
 REDIS_URL = os.getenv("REDIS_URL", "redis://127.0.0.1:6379/0")
 
-POSTGRESQL_DSN = os.getenv("POSTGRESQL_DSN")
+POSTGRESQL_DSN = os.getenv("POSTGRESQL_DSN", "postgresql://main:main@127.0.0.1:5432/main")
```

Comment:
I'd like to rename this to `DATABASE_URL`, as that's a pretty commonly accepted envvar for database connection strings.

(this can be done post merge, not a requirement for you)

Reply:
That's fine, but yeah it would be best on a future PR